src/handwritten/_linux_installation_commands.py

```diff
@@ -0,0 +1,84 @@
+import click
+from typing import List, Dict, Callable
+
+def get_version_ubuntu(dist_version: str) -> str:
+    return dist_version.replace(".", "")
+    
+def get_version_opensuse(dist_version: str) -> str:
+    return dist_version.replace(".", "")
+
+def get_version_rhel(dist_version: str) -> str:
+    return dist_version.split(".")[0]
+
+# Command templates
+apt_install_commands = [
+    ["sudo", "apt", "update"],
+    ["sudo", "apt", "install", "{directory}/NILinux{release}DeviceDrivers/ni-ubuntu{version}-drivers-{release}.deb"],
+    ["sudo", "apt", "update"],
+    ["sudo", "apt", "install", "ni-daqmx"],
+    ["sudo", "dkms", "autoinstall"],
+]
+
+zypper_install_commands = [
+    ["sudo", "zypper", "update"],
+    ["sudo", "zypper", "install", "insserv"],
+    ["sudo", "zypper", "--no-gpg-checks", "install", "{directory}/NILinux{release}DeviceDrivers/ni-opensuse{version}-drivers-{release}.rpm"],
+    ["sudo", "zypper", "refresh"],
+    ["sudo", "zypper", "install", "ni-daqmx"],
+    ["sudo", "dkms", "autoinstall"],
+]
+
+yum_install_commands = [
+    ["sudo", "yum", "update"],
+    ["sudo", "yum", "install", "chkconfig"],
+    ["sudo", "yum", "install", "{directory}/NILinux{release}DeviceDrivers/ni-rhel{version}-drivers-{release}.rpm"],
+    ["sudo", "yum", "install", "ni-daqmx"],
+    ["sudo", "dkms", "autoinstall"],
+]
+
+debian_daqmx_version_command = ["dpkg", "-l", "ni-daqmx"]
+rpm_daqmx_version_command = ["rpm", "-q", "ni-daqmx"]
+
+
+# Mapping of distros to their command templates and version handlers
+linux_commands: Dict[str, Dict[str, Callable]] = {
```

Comment:
`Callable` is short for `Callable[[...], Any]`, which means the function can take any parameters and returns who-knows-what.

To declare the type of each callable, use `typing_extensions.TypedDict` (or `typing.TypedDict`, depending on Python version). Or even better, take advantage of the fact that Python is an object-oriented language and not a dictionary-oriented language, and use a `dataclass` or `typing.NamedTuple` instead.